instructor/migrations/0005_auto_20151030_1112.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
Would it be possible to rename this migration? Something like 0005_removed_order_field would work